code/modules/jobs/job_types/brigphys.dm

```diff
@@ -10,7 +10,7 @@
 	selection_color = "#ffeeee"
 	chat_color = "#b16789"
 	minimal_player_age = 7
-	exp_requirements = 120
+	exp_requirements = 600
```

Comment:
10 hours of playing sec to be a brig phys?

Reply:
10 hours alive as crew, not specifically in sec.

Ideally a brig physician should already have basic medical experience imo, but this still puts it at a lower entry point than the actual sec officers.

Reply:
medical requirement would be better imho

Reply:
Will do.